presto-jdbc/pom.xml

```diff
@@ -142,7 +142,13 @@
         <dependency>
             <groupId>io.airlift</groupId>
             <artifactId>concurrent</artifactId>
-            <scope>test</scope>
+            <exclusions>
```

Comment:
Nice job tracking that one down; thanks!
I don't have strong feelings about excluding vs relocating, so I'll leave it as-is for now.